apps/web-giddh/src/app/vouchers/create/create.component.ts

```
@@ -89,6 +89,9 @@ import { ProformaService } from "../../services/proforma.service";
 import { SettingsProfileActions } from "../../actions/settings/profile/settings.profile.action";
 import { TitleCasePipe } from "@angular/common";
 import { MatSelectChange } from "@angular/material/select";
+import { OcrAction } from "../../ai-ocr/ai-ocr.component";
+import { AiOcrStore } from "../../ai-ocr/utility/ai-ocr.store";
+import { AiOcrService } from "../../services/ai-ocr.service";
 import { EWayBillCreateComponent } from "../../shared/eWayBill/create/e-way-bill-create-component";
 import { ServiceConfig } from "../../services/service.config";
 import { OcrAction } from "../../ai-ocr/ai-ocr.component";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'OcrAction'. Consider to delete it or rename it.

'OcrAction' is defined here:


(lint/suspicious/noRedeclare)